lib/virtfs/xfs.rb

```diff
@@ -1 +1,20 @@
 require "virtfs/xfs/version"
+require_relative "xfs/allocation_group"
+require_relative "xfs/bmap_btree_block"
+require_relative "xfs/bmap_btree_record"
+require_relative "xfs/bmap_btree_root_node"
+require_relative "xfs/directory"
+require_relative "xfs/directory2_data_header"
+require_relative "xfs/directory3_data_header"
+require_relative "xfs/directory_block_tail"
+require_relative "xfs/directory_data_header"
+require_relative "xfs/directory_entry"
+require_relative "xfs/inode"
+require_relative "xfs/inode_map"
+require_relative "xfs/short_form_directory_entry"
+require_relative "xfs/short_form_header"
+require_relative "xfs/superblock"
+
+require_relative "xfs/base"
+require_relative "xfs/file_class"
+require_relative "xfs/dir_class"
```

Comment:
@Fryguy Is there any benefit in using `require` instead of `require_relative`? Shouldn't `require_relative`  be more efficient, since it doesn't have to go through the array of load paths? It seems to me, GEM-internal requires like these are ideal candidates for the use of `require_relative`.